lib/entities/scheduled-action.ts

```diff
@@ -147,8 +147,7 @@ export default function getInstanceMethods(makeRequest: MakeRequest): ScheduledA
       }).then((data) => wrapScheduledAction(makeRequest, data))
     },
     /**
-     * Update current scheduled actions. All changes made to current instance will be saved in the database,
-     * provided the request succeeds.
+     * Update current scheduled actions. All changes made to the current object will be saved, if the request succeeds.
```

Comment:
```suggestion
     * Update the current scheduled action. All changes made to the current object will be saved, if the request succeeds.
```

Perhaps also mention we currently only allow updating the `scheduledFor`